libc-test/build.rs

```diff
@@ -3593,6 +3593,10 @@ fn test_linux(target: &str) {
             // FIXME: The size of `iv` has been changed since Linux v6.0
             // https://github.com/torvalds/linux/commit/94dfc73e7cf4a31da66b8843f0b9283ddd6b8381
             "af_alg_iv" => true,
+
+            // FIXME: Requires >= 5.11 kernel headers
+            "tls12_crypto_info_chacha20_poly1305" if (mips || mips64) && musl => true,
```

Comment:
Did you check there is really no more to add than this ?

Reply:
Yes, I ran `ci/run-docker.sh` for the mips target locally. The issue is only with Chacha20-Poly1305 definitions, so this one struct and the six consts.